coalib/bears/requirements/PackageRequirement.py

```diff
@@ -49,3 +49,38 @@ def check(self):
         :return: Returns True if satisfied, False if not.
         """
         raise NotImplementedError
+
+    @classmethod
+    def multiple(cls, *args):
```

Comment:
nah I don't think this is good. So this class is abstract, i.e. you're not supposed to instantiate it!

If you did something like:

```python
class PackageRequirement:
    ...
    @classmethod
    def multiple(cls, *args):
        manager = cls.manager
        ...
```

and then

```python
class PythonRequirement:
    ...
    manager = 'pip'
```

then you would never run `PackageRequirement.multiple` because that's an abstract class but rather `PythonRequirement.multiple(('perlcritic', '6.2.3'))`